### PR TITLE
[#790] Added support to save map-component coordinates in registration backends

### PR DESCRIPTION
**Pull request by Bartvaderkin**

### CODE REVIEW

File: src/openforms/registrations/contrib/stuf_zds/plugin.py [hunk 1]
```diff
@@ -95,6 +97,13 @@ def _safe_int(num):
             )
 
 
+def _point_coordinate(value):
+    if not value or not isinstance(value, list) or len(value) != 2:
+        return SKIP
+    # cast to str because floats would get localized in the template
```
Comment (sergei-maertens) [inline src/openforms/registrations/contrib/stuf_zds/plugin.py:103]: shouldn't we just disable localization in the template then? https://docs.djangoproject.com/en/2.2/topics/i18n/formatting/#localize

File: src/openforms/registrations/contrib/stuf_zds/plugin.py [hunk 2]
```diff
@@ -110,6 +119,9 @@ class StufZDSRegistration(BasePlugin):
         # "initiator.aanschrijfwijze": FieldConf(RegistrationAttribute.initiator_aanschrijfwijze),
         "initiator.bsn": FieldConf(submission_field="bsn"),
         "initiator.kvk": FieldConf(submission_field="kvk"),
+        "lokatie": FieldConf(
```
Comment (sergei-maertens) [inline src/openforms/registrations/contrib/stuf_zds/plugin.py:122]: is it called `lokatie`? Shouldn't that be `locatie`?

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: src/openforms/registrations/contrib/stuf_zds/plugin.py (hunk 1 above)
Comment (Bartvaderkin) [reply src/openforms/registrations/contrib/stuf_zds/plugin.py]: I forgot about the `unlocalize` filter.

File: src/openforms/registrations/contrib/stuf_zds/plugin.py (hunk 2 above)
Comment (Bartvaderkin) [reply src/openforms/registrations/contrib/stuf_zds/plugin.py]: It is `lokatie` in Stuf but I updated it anyway.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**